### ISSUE TITLE
FromDrawingColor Alpha

### ISSUE THREAD

**Issue by imtrobin** [mariusmuntean/ChartJs.Blazor#118]
Hi, can I suggest to add to ColorUtils

public static string FromDrawingColor (System.Drawing.Color color, float alpha) => ColorUtil.ColorString (color.R, color.G, color.B, alpha);

**Comment by Joelius300** [comment]
Actually, `System.Drawing.Color` already has an [alpha property](https://docs.microsoft.com/en-us/dotnet/api/system.drawing.color.a). We should just expand `FromDrawingColor` to also include the alpha.

Instead of 
```
return ColorHexString(color.R, color.G, color.B);
```
that would be
```
return ColorString(color.R, color.G, color.B, (double)color.A / byte.MaxValue);
```

That should work if I'm not mistaken. We could also consider adding a `ColorHexString` function with alpha included and use that.

Feel free to open a PR for this.

**Comment by Joelius300** [comment]
Closed by ea4b914. Will be released in the next update.
